worker-build/src/main.rs

```diff
@@ -183,3 +188,25 @@ fn write_string_to_file<P: AsRef<Path>>(path: P, contents: String) -> Result<()>
 
     Ok(())
 }
+
+// TODO: Potentially rewrite this to run an actual JavaScript parser.
```

Comment:
If you can get it working in Rust, we use this in the dashboard: https://github.com/guybedford/es-module-lexer

Reply:
This seems interesting, but after a quick glance I don't know if it'd be worth the time to either port it or make some necessary changes and link. I'm fairly confident the current inelegant approach is fine for any code that wasm-bindgen generates today, I mostly wrote that comment as a suggestion if wasm-bindgen makes any changes that would break this in the future.

What do you think @nilslice?